const/meta.ts

```diff
@@ -8,7 +10,8 @@ export const siteConfig = {
     rootIPFS: "https://IPFS-URL-HERE",
     swap: "https://cowswap.exchange",
     docs: "https://docs.cowswap.exchange/",
-    apiDocs: "https://protocol-mainnet.gnosis.io/api/",
+    api: API_BASE_URL + "/mainnet",
+    apiDocs: API_BASE_URL + "/mainnet/",
```

Comment:
Are both supposed to be identical?

Reply:
its dummy, but i did it consciously. One is for the docs and the other is the base url for the API. I could remove the docs and assume is the same if u like it more

Reply:
🤷